manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml

```diff
@@ -0,0 +1,29 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.OpenJDK.11
+Publisher: Microsoft
+PackageName: Microsoft OpenJDK 11 Preview
```

Comment:
The name must be `Microsoft Build of OpenJDK 11 (Preview)`

Reply:
Fixed.